processor/stash/stash.go

```diff
@@ -33,11 +33,11 @@ func init() {
 }
 
 func loadConfig() {
-	errorStashEnabled = config.GetBool("Processor.errorStashEnabled", true)
-	errReadLoopSleep = config.GetDuration("Processor.errReadLoopSleepInS", time.Duration(30)) * time.Second
-	errDBReadBatchSize = config.GetInt("Processor.errDBReadBatchSize", 1000)
-	noOfErrStashWorkers = config.GetInt("Processor.noOfErrStashWorkers", 2)
-	maxFailedCountForErrJob = config.GetInt("Processor.maxFailedCountForErrJob", 3)
+	config.RegisterBoolConfigVariable("Processor.errorStashEnabled", false, &errorStashEnabled, true)
+	config.RegisterDurationConfigVariable("Processor.errReadLoopSleepInS", time.Duration(30), &errReadLoopSleep, true, time.Second)
+	config.RegisterIntConfigVariable("Processor.errDBReadBatchSize", 1000, &errDBReadBatchSize, true, 1)
+	config.RegisterIntConfigVariable("Processor.noOfErrStashWorkers", 2, &noOfErrStashWorkers, true, 1)
+	config.RegisterIntConfigVariable("BatchRouter.maxFailedCountForErrJob", 3, &maxFailedCountForErrJob, true, 1)
```

Comment:
Wrong key used here? BatchRouter.maxFailedCountForErrJob  should be Processor.maxFailedCountForErrJob ?